src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -146,9 +146,9 @@ endpoint {{endpointId}} {
     ram      {{!align~}}
         {{~/if~}}
         attribute {{asLowerCamelCase name}}
-        {{~#if_is_non_zero_default defaultValue~}}
-           {{!need space}} default = {{#if (isString type)}}"{{defaultValue}}"{{else}}{{defaultValue}}{{/if}}
-        {{~/if_is_non_zero_default~}}
+        {{~#if defaultValue~}} 
```

Comment:
So this works because if the default is 0 then `defaultValue` is `"0"` (a string) not `0` (a number)?

This is probably OK, but I don't know how valid that assumption is in general..

Reply:
I could not find a helper here. Agree that it relies on things that are not necesarely stable.

Will merge with the assumption "this works better than before" however it may need updates if we ever store numeric instead of string.